Gopkg.toml

```diff
@@ -52,3 +52,7 @@ required = [
 [[override]]
   name = "k8s.io/gengo"
   revision = "b58fc7edb82e0c6ffc9b8aef61813c7261b785d4"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/hashicorp/go-version"
```

Comment:
Don't think this needs to be added as a constraint given we are constraining it to master anyway. A simple `dep ensure` without adding this block should yield the same result, sans this change

Reply:
Done.